Apps/Database/src/Delegates/DBProfileDelegate.cs

```diff
@@ -160,10 +160,10 @@ public int GeUnregisteredInvitedUsersCount()
         /// <inheritdoc />
         public int GetTodayLoggedInUsersCount(TimeZoneInfo timezone)
         {
-            DateTime startDate = TimeZoneInfo.ConvertTimeFromUtc(DateTime.UtcNow.Date, timezone);
+            DateTime startDate = TimeZoneInfo.ConvertTimeFromUtc(DateTime.Today.ToUniversalTime(), timezone);
             int result = this.dbContext.UserProfile
                 .Count(u => u.LastLoginDateTime.HasValue &&
-                TimeZoneInfo.ConvertTimeFromUtc(u.LastLoginDateTime.Value, timezone) >= startDate);
+               u.LastLoginDateTime.Value >= startDate.ToUniversalTime());
```

Comment:
what is the point of converting from utc, to convert back to utc here? Also, the internal date needs to be in the local timezone

Reply:
see discussion in rocket chat